src/transformers/generation/logits_process.py

```diff
@@ -193,12 +193,49 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class RepetitionPenaltyLogitsProcessor(LogitsProcessor):
     r"""
-    [`LogitsProcessor`] enforcing an exponential penalty on repeated sequences.
+    [`LogitsProcessor`] prevents the repetition of previous tokens through an exponential penalty. 
+    This technique invloves adjusting the scores assigned to previously generated tokens, discouraging repetition. 
+    This technique shares some similarities with coverage mechanisms and other aimed at reducing repetition. 
+    During the text generation process, the probability distribution for the next token is determined using a formula that incorporates 
+    token scores based on their occurrence in the generated sequence. Tokens with higher scores are less likely to be selected. 
+    The formula takes into account a fair bunch of variables which can be further seen in the [paper](https://arxiv.org/pdf/1909.05858.pdf).
+
+    <Tip>
+
+    It is worth noting that the effectiveness of penalized sampling relies on the model's ability to learn a reliable distribution of 
+    tokens during its training.
+
+    </Tip>
```

Comment:
```suggestion
```
(the same applies to all logits processors -- they all rely in good baseline model performance)